packages/manager/modules/pci/src/projects/project/creating/translations/Messages_fr_FR.json

```diff
@@ -3,10 +3,6 @@
   "pci_projects_project_creating_secondary_title": "La création de votre projet est en cours",
   "pci_projects_project_creating_explain1": "Pour finaliser la création de votre projet, veuillez effectuer le paiement du bon de commande <a href=\"{{ href }}\">{{ orderId }}</a>.",
   "pci_projects_project_creating_explain2": "Si cette page est toujours présente plusieurs minutes après le paiement du bon de commande, merci de contacter <a href=\"{{ href }}\">notre service client</a>.",
-  "pci_projects_project_creating_not_paid_title": "En route pour l'aventure Public Cloud",
```

Comment:
It seems that this translation key is still used.

Are you sure about removing `pci_projects_project_creating_not_paid_title`?

Please refer to this following file: https://github.com/ovh-ux/manager/pull/985/files#diff-19dfc170661c5c66ec2e6b4028bf87dfR18

Reply:
Don't worry, there are present ;-) https://github.com/ovh-ux/manager/blob/master/packages/manager/modules/pci/src/projects/project/creating/notPaid/translations/Messages_fr_FR.json#L2